lxd-qemu-snap/snapcraft.yaml

```diff
@@ -56,7 +56,7 @@ parts:
 
   virgl:
     source: https://gitlab.freedesktop.org/virgl/virglrenderer.git
-    source-tag: virglrenderer-1.0.1
+    source-tag: virglrenderer-1.1.0
```

Comment:
I don't remember if it was possible to use `source-commit` for this part? If so, could you use `1aeaf5e10a9c89096e96d09599aa419d5c50712f` which belongs to that tag?

Reply:
To be honest I don't know why we are using `source-commit` thing in there. We don't distribute this snap and never upload it to the store. It's more convenient to have explicit tag/branch references in this case, IMHO.

Reply:
For some reason, Tom changed it to use `source-commit` thing in https://github.com/canonical/lxd-pkg-snap/commit/d1ec45f6ce657bedfe52ce450c621853b4abd9f4 commit.

Reply:
I do agree that `source-tag` is more convenient but tags are mutable meaning we don't have guaranties that we are getting the same bits from one build to the other.

This snap being an unpublished one indeed changes how we need to be strict about it. I'm fine either way which is why I approved it as-is ;)

Reply:
I'm not against going with `source-commit` thing. Just decided to mention a convenience of `source-tag` ;-)